cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceHandler.java

```diff
@@ -216,9 +297,7 @@ public void update(HttpRequest request, HttpResponder responder, @PathParam("nam
     Id.DatasetInstance datasetInstance = Id.DatasetInstance.from(namespaceId, name);
     disableExplore(datasetInstance);
 
-    if (!createDatasetInstance(creationProperties, namespaceId, name, responder, "update")) {
```

Comment:
Ignore the above comment. I see that boolean return was changed to throwing exception.